nucypher/policy/kits.py

```diff
@@ -122,7 +122,7 @@ def _payload(self) -> bytes:
 
     @classmethod
     def _brand(cls) -> bytes:
-        return b'MK'
+        return b'MK  '
```

Comment:
I'm ok with spaces either way, but just throwing it out there that underscores could be used instead -it would make the 4-bytes a little more obvious, and it is also already part of the `\w` match for regex.

Reply:
Yes, fair enough, we can just use a single `\w`. Fixed.